### PR TITLE
Update aptos.toml

### PR DESCRIPTION
**Pull request by 0xneil**
adding additional aptos ecosystem projects (https://github.com/aptos-foundation/ecosystem-projects)

### CODE REVIEW

**Review by jckail — APPROVED** [review]
Well done!

**Review by jubos — CHANGES_REQUESTED** [review]
This is a good start!

It will need some addition of the toml files for the individual sub ecosystems to land. 
Please look at #303 and #282 as examples of what the Moonbeam team did.  It makes the large diffs much easier to review.

Separately, the sub ecosystems are only "primary" sub ecosystems.  Most of these look like Aptos or Move subecosystems but projects like Celer are cross chain and are "supporting" only.  We don't have the supporting sub ecosystems attribute exposed in this taxonomy yet, but we will have it in the next year.

A few good rules of thumb for identifying sub ecosystems of Aptos:
1/ if the governance actions and/or governance token originates on the parent protocol, it is likely a sub ecosystem.
2/ if you go to the project's web page and it only mentions the parent ecosystem, it is likely a sub ecosystem.
3/ If the parent has a domain specific language, and the potential sub is implemented in it, then it is likely a sub ecosystem.  

Please let us know if you have any questions.

**Comment by jckail** [comment]
Hey @jubos ! :) 

Really appreciate your help here, "Move" would not be considered an ecosystem by our interpretation, its a programming language not an ecosystem,  all of the sub ecosystems @0xneil included *SHOULD* meet the criteria of 1,2,3  -- could you help us understand where you think these don't meet the 3 criteria? 

Could you potentially link us to or share with us your definitions for your taxonomy? 

REALLY REALLY appreciate your help here :)

**Comment by jckail** [comment]
Please help us understand what exactly needs to be done to get the correct attributions to aptos :)

**Comment by jckail** [comment]
@jckail

**Comment by 0xneil** [comment]
thanks for the feedback @jubos - made a few edits.

File: data/ecosystems/k/koizumi.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Ecosystem Level Information
+title = "Koizumi"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/KoizumiAPT]
```
Comment (gregnazario) [inline data/ecosystems/k/koizumi.toml:6]: This needs to be quoted

File: data/ecosystems/p/patronusfi.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Ecosystem Level Information
+title = "Patronusfi"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/patronusfi]
```
Comment (gregnazario) [inline data/ecosystems/p/patronusfi.toml:6]: This needs to be quoted

File: data/ecosystems/s/snotratech.toml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Ecosystem Level Information
+title = "SnotraTech"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/snotratech]
```
Comment (gregnazario) [inline data/ecosystems/s/snotratech.toml:6]: This needs to be quoted

File: data/ecosystems/u/umi-ag.toml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Ecosystem Level Information
+title = "Umi"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/umi-ag]
```
Comment (gregnazario) [inline data/ecosystems/u/umi-ag.toml:6]: This needs to be quoted

**Review by gregnazario — COMMENTED** [review]
Just some toml format issues for you @0xneil

**Pull request closed without merging**